### DISCUSSION TITLE
The purpose of the discussion?

### DISCUSSION THREAD

**Discussion by yarikoptic** [dandi/helpdesk#3] [Q&A]
as opposed to issues/wiki/projects?

**Comment by satra** [comment]
mostly because of a different ui, with voting and discussion. wiki and projects do not have that.

**Reply by bendichter** [reply]
I'm really impressed with the Discussion interface! It's like a focused stack overflow, integrated directly into GitHub. Very cool. The only problem is that it's a new feature that users may not know about and discoverability isn't great, so I'll try to help them out in the README of this repo.
